Dockerfile.builder.multi_arch

```diff
@@ -6,8 +6,14 @@
 #
 # Using cilium-builder as the base to ensure libc etc. are in sync.
 #
-FROM quay.io/cilium/cilium-builder:2020-02-26 as builder
+# The base image here should be multi-arched beforehand when used for non-amd64
+# platforms, e.g, arm64 platform.
```

Comment:
Please leave the `Dockerfile.builder` untouched for now as we will transition from one to the other slowly. In the end it is intended that this `Dockerfile.builder.multi_arch` will replace the `Dockerfile.builder` but for now it is important that we don't brake existing builds on docker hub.